deals/rpc/rpc.go

```diff
@@ -139,8 +139,12 @@ func (s *RPC) Watch(req *WatchRequest, srv RPCService_WatchServer) error {
 	}
 
 	for update := range ch {
+		var strProposalCid string
+		if update.ProposalCid.Defined() {
+			strProposalCid = update.ProposalCid.String()
+		}
```

Comment:
Everyone returning a stringified version, should take care of returning empty for coherence.
The client will have the safety-net mentioned in the above comment, but try to be coherent on returning empty for undefined.